### PR TITLE
add some sanity checks for CI/Build MINGW

### PR DESCRIPTION
**Pull request by adrpo**
- make sure the compiler can build and simulate a simple model with C, C++ and that an FMU can be build
- check that linearization works (for matlab)
- check that OMEdit can start (not missing any dlls)

### CODE REVIEW

**Comment by sjoelund** [comment]
Shouldn't you use the MINGW and OSX labels for this job?

**Comment by adrpo** [comment]
Ouch, yes, just MINGW.

**Comment by adrpo** [comment]
Thanks for the heads up! I now restarted the PR.

File: .CI/common.groovy [hunk 1]
```diff
@@ -178,6 +178,24 @@ void buildOMC(CC, CXX, extraFlags) {
      echo make -f 'Makefile.omdev.mingw' \${MAKETHREADS} BUILDTYPE=Release runtimeCPPmsvcinstall
      echo cd \${MSYS_WORKSPACE}
      echo make -f 'Makefile.omdev.mingw' \${MAKETHREADS} BUILDTYPE=Release runtimeCPPinstall
+     echo echo Check that omc can be started and a model can be build for NF OF with runtimes C Cpp FMU
```
Comment (adeas31) [inline .CI/common.groovy:181]: Do you actually need `echo` twice here?

Comment (sjoelund) [reply .CI/common.groovy:181]: That will make it echo when you run the script later

**Pull request merged**